core/src/main/java/jenkins/tools/ToolConfigurationCategory.java

```diff
@@ -0,0 +1,21 @@
+package jenkins.tools;
```

Comment:
This comment was supposed to go on line 12(ish). The idea is to prevent plugins from declaring whatever Descriptor they have to be in the tools category (like we do for `GlobalMavenConfig`).